gemfiles/rack_3.gemfile

```diff
@@ -0,0 +1,8 @@
+# This file was generated by Appraisal
```

Comment:
Normally this file is autogenerated by adding a new entry to `Appraisals` file and running `appraisal install`.

Reply:
Done in https://github.com/rack/rack-attack/pull/586/commits/5d3da4975a8c8d424ef84acdf672684ff9ba04ab.